src/fairness/weighted_tree/Makefile.am

```diff
@@ -42,3 +42,5 @@ libweighted_tree_la_CXXFLAGS = \
     $(CODE_COVERAGE_CFLAGS) \
     $(SQLITE_CFLAGS) \
     $(AM_CXXFLAGS)
+
+EXTRA_DIST = libweighted_tree.la
```

Comment:
Hm, `libweighted_tree.la` should be a product of the build and not a source that needs to be packed up with the distribution. What led you to adding this file to `EXTRA_DIST`?

Reply:
Thanks for pointing this out. I wasn't sure myself why `make distcheck` wasn't working. If I leave out the `EXTRA_DIST` assignment, this is the error that gets generated:

```
make[6]: *** No rule to make target '../../src/fairness/weighted_tree/libweighted_tree.la', needed by 'flux-account-update-fshare'.  Stop.
```